components/index.js

```diff
@@ -3,6 +3,8 @@ export { default as Dashicon } from './dashicon';
 export { default as FormToggle } from './form-toggle';
 export { default as IconButton } from './icon-button';
 export { default as Panel } from './panel';
+export { default as PanelHeader } from './panel/header';
+export { default as PanelBody } from './panel/body';
```

Comment:
Thanks for context @nylen . I'm finding I'm reading the sequence of these pull requests very much out of order 😛